### ISSUE TITLE
Showing multiple hit moves' animation as many times it hits

### ISSUE THREAD

**Issue by Lord-Haji** [Zarel/Pokemon-Showdown-Client#748]

**Comment by Ridaz** [comment]
From what I've heard, this was a thing in the past. The reason it was removed was because it would make the turn last a lot longer

**Comment by Zarel** [comment]
Yeah, the games also make them only hit once. tbh I've been planning on a way to do it but quickly, but then I forgot about it amidst all the other crises I've had to deal with...
